### ISSUE TITLE
Lemon market: unhashable type: 'Currency'

### ISSUE THREAD

**Issue by Toub** [oTree-org/oTree#53]
I got the following error running Lemon market game with default configuration, from master branch, on display result page: 

```
TypeError at /p/hefufuzu/lemon_market/Results/8/

unhashable type: 'Currency'

Request Method: 	GET
Request URL: 	http://otree.localhost/p/hefufuzu/lemon_market/Results/8/
Django Version: 	1.8.8
Exception Type: 	TypeError
Exception Value: 	

unhashable type: 'Currency'

Exception Location: 	/usr/local/lib/python3.5/site-packages/django/db/models/base.py in _get_FIELD_display, line 902
Python Executable: 	/usr/local/bin/python3.5
Python Version: 	3.5.1
Python Path: 	

['/app/sources/games',
 '/usr/local/bin',
 '/app/sources/core',
 '/usr/local/lib/python35.zip',
 '/usr/local/lib/python3.5',
 '/usr/local/lib/python3.5/plat-linux',
 '/usr/local/lib/python3.5/lib-dynload',
 '/root/.local/lib/python3.5/site-packages',
 '/usr/local/lib/python3.5/site-packages',
 '/app/sources/games']

Server time: 	Mon, 18 Apr 2016 17:41:11 +0000

```

Is this game broken?

**Comment by oTree-org** [comment]
I have a pull request to our dependency to fix this: https://github.com/Suor/django-easymoney/pull/7

**Comment by oTree-org** [comment]
Fixed in 0.4.48
